data/default_os_images.json

```
@@ -232,5 +232,29 @@
         "cpu_architecture": "s390x",
         "url": "https://mirror.openshift.com/pub/openshift-v4/s390x/dependencies/rhcos/pre-release/4.19.0-ec.5/rhcos-4.19.0-ec.5-s390x-live-iso.s390x.iso",
         "version": "9.6.20250402-0"
+    },
+    {
+        "openshift_version": "4.20",
+        "cpu_architecture": "x86_64",
+        "url": "https://mirror.openshift.com/pub/openshift-v4/x86_64/dependencies/rhcos/pre-release/4.20.0-ec.2/rhcos-4.20.0-ec.2-x86_64-live-iso.x86_64.iso",
+        "version": "9.6.20250402-0"
+    },
+    {
+        "openshift_version": "4.20",
+        "cpu_architecture": "arm64",
+        "url": "https://mirror.openshift.com/pub/openshift-v4/aarch64/dependencies/rhcos/pre-release/4.20.0-ec.2/rhcos-4.20.0-ec.2-aarch64-live-iso.aarch64.iso",
+        "version": "9.6.20250402-0"
+    },
+    {
+        "openshift_version": "4.20",
+        "cpu_architecture": "ppc64le",
+        "url": "https://mirror.openshift.com/pub/openshift-v4/ppc64le/dependencies/rhcos/pre-release/4.20.0-ec.2/rhcos-4.20.0-ec.2-ppc64le-live-iso.ppc64le.iso",
+        "version": "9.6.20250402-0"
+    },
+    {
+        "openshift_version": "4.20",
+        "cpu_architecture": "s390x",
+        "url": "https://mirror.openshift.com/pub/openshift-v4/s390x/dependencies/rhcos/pre-release/4.20.0-ec.2/rhcos-4.20.0-ec.2-s390x-live-iso.s390x.iso",
+        "version": "9.6.20250402-0"
```

```suggestion
        "version": "9.6.20250513-0"
```

here and everywhere elase